controls/SV-204458.rb

```diff
@@ -17,31 +16,41 @@
     If the release is not supported by the vendor, this is a finding.'
   desc 'fix', 'Upgrade to a supported version of the operating system.'
   impact 0.7
-  tag 'legacy': ['SV-86621', 'V-71997']
-  tag 'severity': 'high'
-  tag 'gtitle': 'SRG-OS-000480-GPOS-00227'
-  tag 'gid': 'V-204458'
-  tag 'rid': 'SV-204458r744100_rule'
-  tag 'stig_id': 'RHEL-07-020250'
-  tag 'fix_id': 'F-4582r462547_fix'
-  tag 'cci': ['CCI-000366']
+  tag legacy: ['SV-86621', 'V-71997']
+  tag severity: 'high'
+  tag gtitle: 'SRG-OS-000480-GPOS-00227'
+  tag gid: 'V-204458'
+  tag rid: 'SV-204458r744100_rule'
+  tag stig_id: 'RHEL-07-020250'
+  tag fix_id: 'F-4582r462547_fix'
+  tag cci: ['CCI-000366']
   tag nist: ['CM-6 b']
-  tag subsystems: ["redhat_release"]
-  tag 'host', 'container'
+  tag subsystems: ['redhat_release']
+  tag 'host'
+  tag 'container'
 
   release = os.release
+  supported_releases = ['7.6', '7.7', '7.9']
```

Comment:
BUG: does not account for os.release that comes back as "7.9.2009". This passed in main, fails now.
![image](https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/assets/34140975/68a1da65-835e-4109-8812-88501c2f060e)

Reply:
I pushed a change for this. Feel free to test again when it is convenient for you.

Reply:
Fixed Thanks!
![image](https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/assets/34140975/56e2b4ad-f3fa-4932-8bdf-4fbbf36c9159)